### ISSUE TITLE
Bug-Check after Meltdown update for 15063

### ISSUE THREAD

**Issue by CaptainBigSmoke** [DarthTon/Blackbone#228]
Hi there,
after the January 3 update KB4056891 (i.e. Meltdown fix) for the 15063 kernel I have been hit with bugchecks.

After digging I found that the problem is that the DYN_PDE_BASE and DYN_PTE_BASE
offsets are outdated. The correct ones for 15063 after the hotfix (i.e. patchlevel 15063.850) are as follows:

```
else if (pData->ver >= WINVER_10_CU)
        {
            pData->DYN_PDE_BASE = *(PULONG_PTR)(pMiGetPhysicalAddress + 0x4B + 2);
            pData->DYN_PTE_BASE = *(PULONG_PTR)(pMiGetPhysicalAddress + 0x58 + 2);
        }
```

Weirdly Blackbone still reports the os version as "BlackBone: OS version 10.0.15063.0.0 - 0xa00". Thus
it is not really easily possible to add some if condition to make this backwards compatible. I guess one needs to additionally read something else to see if we have a patched os.

Apparently 16299 has not been yet served this fix but I assume the offset will also change there.

Cheers,
CaptainBigSmoke

**Comment by DarthTon** [comment]
Are you sure these offsets are for 15063. My fixed 16299.192 has exactly same values now.

**Comment by CaptainBigSmoke** [comment]
Yep, see yourself: 
![img](https://user-images.githubusercontent.com/4469708/34633015-83f13ece-f279-11e7-9f7e-fc55432bdde2.jpg)
